manifest.jsonnet

```diff
@@ -46,7 +46,7 @@ else
 
 local INDEX_MANAGER_FILES = ['core/storage.js', 'index-manager.js'];
 json.addIcons(icons())
-.addPermissions(['storage', 'unlimitedStorage'])
+.addPermissions(['storage', 'unlimitedStorage', 'webRequest', 'webRequestBlocking', '*://crates.io/api/v1/crates/*'])
```

Comment:
We use Manifest V3 already, after loading into Chrome, it said: `webRequestBlocking' requires manifest version of 2 or lower`. Actually, I think we can remove those permissions.